soc/intel/intel_adsp/ace/power.c

```diff
@@ -334,25 +334,26 @@ void pm_state_set(enum pm_state state, uint8_t substate_id)
 			/* do power down - this function won't return */
 			ret = pm_device_runtime_put(INTEL_ADSP_HST_DOMAIN_DEV);
 			__ASSERT_NO_MSG(ret == 0);
-			power_down(true, uncache_to_cache(&hpsram_mask),
+			power_down(true, sys_cache_cached_ptr_get(&hpsram_mask),
```

Comment:
@ceolin , @lyakh this fails a sparse check:

https://github.com/thesofproject/sof/actions/runs/9253499236/job/25453295192?pr=9168

```
zephyr/soc/intel/intel_adsp/ace/power.c:337:66: warning:
    incorrect type in argument 2 (different address spaces)
```